src/index.js

```diff
@@ -13,6 +13,12 @@ const multiaddr = require('multiaddr')
 const once = require('once')
 const request = require('request')
 
+function noop () {}
+
+function cleanMultiaddr (ma) {
+  return ma.decapsulate('/p2p-webrtc-direct')
+}
```

Comment:
This went from a very complicated:

```
function cleanMultiaddr (ma) {		
  return multiaddr(ma.toString().split('/').slice(2).join('/'))		
}
```

To the above. Who would know that using multiaddrs as they are designed to be used would simplify things :) 

//cc @lgierth